configuration/device-tree.md

```diff
@@ -547,24 +611,24 @@ fragment@0 {
 
 Any overlays using the numeric variants will be modified to use the new aliases.
 
-<a name="part3.4"></a>
 ### 3.4: HATs and Device Tree
+<a name="part3.4"></a>
 
-A Raspberry Pi HAT is an add-on board for a "Plus"-shaped (A+, B+ or Pi 2 B) Raspberry Pi with an embedded EEPROM. The EEPROM includes any DT overlay required to enable the board, and this overlay can also expose parameters.
+A Raspberry Pi HAT is an add-on board for a "Plus"-shaped (A+, B+, Pi 2, Pi 3 and Pi 4) Raspberry Pi, with an embedded EEPROM. The EEPROM includes any DT overlay required to enable the board (or the name of an overlay to load from the filing system), and this overlay can also expose parameters.
```

Comment:
Did the pHAT for the Zero ever get standardised? I'm assuming that'll have the EEPROM on as well.

Reply:
IIRC it was called a uHAT in the TV-tuner addon? I think pHAT is a Pimoroni thing?
_...searches..._
https://www.raspberrypi.org/blog/raspberry-pi-tv-hat/

Reply:
Yes - it's just a mechanical difference. I'll change it to "add-on board with an embedded EEPROM designed for a Raspberry Pi with a 40-pin header".

Reply:
Nice and short! :-/